kingpin/actors/base.py

```diff
@@ -308,6 +308,15 @@ def _fill_in_contexts(self, context={}, strict=True):
         # Finally, convert the string back into a dict and store it.
         self._options = json.loads(new_options_string)
 
+    def get_orgchart(self, parent=''):
```

Comment:
It would be great in the long run if the there was some kind of a `status` object associated with every `BaseActor`. This object could include information like ...

 * The `parent`/`child` relationship
 * Execution time of the actor
 * Tracking # of API calls potentially (lets say, if the actor did was able to do this)
 * Return values/stored data from the actor (this could go hand in hand with issue #173)